src/reuse/modules/util/browser.ts

```diff
@@ -417,6 +417,45 @@ export class Browser {
 
     return false;
   }
-}
 
+  /**
+   * @function isMobile
+   * @memberOf util.browser
+   * @description Indicates a mobile session
+   * @returns {boolean} Return true if its a mobile session driver.
+   * @example await util.browser.isMobile();
+   */
+  async isMobile(): Promise<boolean> {
+    const vl = this.vlf.initLog(this.isMobile);
+    vl.log('Indicates is a mobile session? or browser session');
+    return browser.isMobile();
+  }
+
+   /**
+   * @function isAndroid
+   * @memberOf util.browser
+   * @description Indicates a mobile session
+   * @returns {boolean} Return true if its a Android session driver.
+   * @example await util.browser.isAndroid();
+   */
+   async isAndroid(): Promise<boolean> {
+    const vl = this.vlf.initLog(this.isAndroid);
+    vl.log('Indicates is a Android session? or iOS session');
+    return browser.isAndroid();
+  }
+
+  /**
+   * @function isiOS
+   * @memberOf util.browser
+   * @description Indicates an iOS session
+   * @returns {boolean} Return true if its a iOS session driver.
+   * @example await util.browser.isiOS();
+   */
+  async isiOS(): Promise<boolean> {
```

Comment:
@Templada Previous comment #287 
Would go with strict camelCase here - @cvakiitho @vlad9719 wdy think?

Reply:
As per the Apple standard of using this name as iOS(i-small letter, OS -would be capital) now it's changed to as per this standard.

Reply:
The version from Siri is fine for me

Reply:
as I remember I agreed with Ben that we wanted to have true camel-case.
Per reference we also went same approach with `json` in [function names](https://github.com/SAP/wdio-qmate-service/blob/a189ae33828c0c7351b7246b1c06f20d9dcd08ca/src/scripts/dataExchange/dataExchangeUtil.ts#L62)

Reply:
Yes I know that is a complicated topic and there are different opinions on that. In general, there is no right ore wrong.

However, as Marvin said, using strict camel case even for terms like "OS" is the only way to separate the words correctly in a consistent way. 
Imagine you have something like "get OS info", if you would keep the OS, it is not clear where the separation happens -> "getOSInfo"

Reply:
Changed as per detailed review comments.